build/musicbrainz-dev/scripts/fetch-dump.sh

```diff
@@ -127,6 +127,7 @@ then
 	"${WGET_CMD[@]}" -nd -nH -c -r -P "$SEARCH_DUMP_DIR" \
 		--accept 'MD5SUMS,*.tar.zst' --no-parent --relative \
 		"${BASE_DOWNLOAD_URL}/data/search-indexes/$DUMP_TIMESTAMP/"
+	echo "$(date): Checking MD5 sums..."
```

Comment:
AFAICT this one does not OK on completion?

Reply:
On the next line, the command `md5sum` does _OK_ on completion, but it says neither that it started and nor what _OK_ is about for each file.